fs/pathFunctions.go

```diff
@@ -0,0 +1,49 @@
+package fs
```

Comment:
Don't call the file "pathFunctions.go". It should be named based on the contents ("path.go"), but since it's just one file and not all the contents are actually strictly path-related, the expectation would be to name it after the package. So call it "fs.go" for now.